zipline/zipline_deprecation_warning.py

```diff
@@ -13,20 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cpython cimport bool
```

Comment:
Just confirming this means `live_bar_data.pyx` is removed in the corresponding internal patch?